drivers/hmis/lib/form_data/records/funder.json

```diff
@@ -42,30 +61,51 @@
       "type": "GROUP",
       "link_id": "dates",
       "text": "Operating Period",
-      "required": false,
-      "display": { "direction": "row" },
       "item": [
+        {
+          "link_id": "date-range-warning",
+          "type": "DISPLAY",
+          "component": "WARNING_ALERT",
+          "text": "Date range does not overlap with the project operating period.",
+          "enable_behavior": "ANY",
+          "enable_when": [
+            {
+              "question": "end",
+              "operator": "LESS_THAN",
+              "compare_question": "project-start"
+            },
+            {
+              "question": "start",
+              "operator": "GREATER_THAN",
+              "compare_question": "project-end"
+            }
+          ]
+        },
```

Comment:
This is my solution for this spec:
> funder start and end dates can be outside of project, but we should warn if there’s no overlap

And this is how it looks. The alert appears and disappears as soon as you change the date input values.
<img width="894" alt="Screen Shot 2022-12-01 at 2 14 31 PM" src="https://user-images.githubusercontent.com/5333982/205139896-e3ac3e3d-b3e1-46e8-8e51-4d0e4f312c69.png">


It's not ideal because it doesn't state the project start and end dates within the alert, but as long as we show the project operating period elsewhere on the page, I think its ok.

Reply:
I think this is great!